build/docker/Dockerfile

```diff
@@ -65,6 +65,7 @@ COPY build/docker/entrypoint.sh /app/
 
 ENV WORKDIR=/app
 ENV DUMB_INIT_SETSID=0
+ENV SERVICE=${SERVICE}
```

Comment:
This has been renamed to COMPONENT. You may want to align the names here and elsewhere. Also, update readme on how to run a specific service.